src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User verifiableUser = UserService.findByEmail(email);
+        if (verifiableUser != null) {
+            return verifiableUser.getPassword().equals(password);
+        }
         return false;
```

Comment:
this code can be written in one line

Reply:
How it can be written in one line. You mean like this:
if (verifiableUser != null) return verifiableUser.getPassword().equals(password);

But what about Java Guide Style requirement then?